ui/views.py

```diff
@@ -241,24 +243,50 @@ def extra_context(self):
                 )
             qs_prefix = "?{0}&".format("&".join(qs_prefix))
 
+        def make_dict(result):
+            """Serialize result to dict."""
+            return {
+                "lid": result.lid,
+                "resource_type": result.resource_type,
+                "title": result.title,
+                "course": result.course,
+                "run": result.run,
+                "description": result.description,
+                "description_path": result.description_path,
+                "preview_url": result.preview_url,
+                "xa_nr_views": result.nr_views,
+                "xa_nr_attempts": result.nr_attempts,
+                "xa_avg_grade": result.avg_grade,
```

Comment:
@ShawnMilo indicated that he was going to change it soon and I wanted to save the effort of doing it twice